### DISCUSSION TITLE
Claude project files

### DISCUSSION THREAD

**Discussion by davidmoshal** [AllYourBot/hostedgpt#461] [General]
Hi, awesome project, thanks for creating it.
I'm wondering if there are any plans to support Claude project files?

**Comment by keithschacht** [comment]
Hi @davidmoshal, yes, we are adding support for that. Currently the feature is called “Assistants” rather than projects. We aren’t fully there yet but we’re getting close.

If you click your name in the bottom left, you can go to settings. Within settings, you can create a new assistant and you can set custom instructions and a name for this new assistant. We are just starting on the feature for you to be able to upload documents to this assistant. I think that’s the most important missing feature. But we also need to create a concept of a team so that you can make this assistant visible to certain people (or even get a publicly shareable link to the assistant). This feature hasn’t been started yet but it’s much easier than the document one so it shouldn’t be too tricky.
